From e1ca562a7e324885fa14dd58651bc622ebd6f9cc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Juho=20H=C3=A4m=C3=A4l=C3=A4inen?=
 <juho.hamalainen@jolla.com>
Date: Mon, 23 Mar 2020 17:40:04 +0200
Subject: [PATCH 40/40] (hybris) Allow input group to use vibrator.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

[hybris] Allow input group to use vibrator. JB#49350

Signed-off-by: Juho Hämäläinen <juho.hamalainen@jolla.com>
---
 rootdir/init.rc | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index c5a6a0bc0..b2f6540ab 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -649,10 +649,11 @@ on boot
     chown system system /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq
     chmod 0660 /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq
 
+    # SailfishOS: allow input group to use vibrator
+    chown system input /sys/class/leds/vibrator/activate
+    chown system input /sys/class/leds/vibrator/duration
     chown system system /sys/class/leds/vibrator/trigger
-    chown system system /sys/class/leds/vibrator/activate
     chown system system /sys/class/leds/vibrator/brightness
-    chown system system /sys/class/leds/vibrator/duration
     chown system system /sys/class/leds/vibrator/state
     chown system system /sys/class/timed_output/vibrator/enable
     chown system system /sys/class/leds/keyboard-backlight/brightness
-- 
2.20.1

